src/VolleyManagement.Backend/VolleyManagement.Services/TournamentService.cs

```diff
@@ -36,6 +36,7 @@ public class TournamentService : ITournamentService
         private const int MIN_TEAMS_TO_PLAY_ONE_ROUND = 2;
         private const int GAMES_TO_PLAY_ONE_ROUND = 2;
         private const int DONT_CREATE_SCHEDULE_TEAMS_COUNT = 1;
+        private const int FIRST_ROUND = 1;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'FIRST_ROUND'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1144)

Reply:
I ❤️ Sonar))))